presto-hive/src/main/java/com/facebook/presto/hive/ParquetHiveRecordCursor.java

```diff
@@ -808,6 +805,26 @@ public ParquetListConverter(Type prestoType, String columnName, GroupType listTy
             }
         }
 
+        //copied over from Apache Hive
+        private boolean isElementType(parquet.schema.Type repeatedType, String parentName)
+        {
+            if (repeatedType.isPrimitive() ||
+                    (repeatedType.asGroupType().getFieldCount() != 1)) {
```

Comment:
the code used to check `> 1` instead of `!= 1` should we really be allowing `getFieldCount() == 0`?